examples/blog/server.js

```diff
@@ -17,7 +17,7 @@ async function main () {
 
   const app = express()
   app.locals.store = new ResourceStore({ quadStore: store })
-  app.use(hydraBox(api, store))
+  app.use(hydraBox(api, { store }))
```

Comment:
Unrelated, but I would rename `locals.store` to prevent confusion between that and the loader's store